src/core/client/stream/tabs/Comments/Comment/CommentContainer.tsx

```diff
@@ -421,7 +442,21 @@ export const CommentContainer: FunctionComponent<Props> = ({
     comment.lastViewerAction !== "CREATE" &&
     comment.lastViewerAction !== "EDIT";
 
-  return (
+    if(comment.body === 'DELETED'){
```

Comment:
I see you have used the string constant of `DELETED` in multiple places. Would you mind extracting it into a constant variable or named enum?